Gemfile

```diff
@@ -20,18 +20,10 @@ group :test do
     gem 'rspec', '~> 3.0'
     gem 'json'
   end
-
-  if RUBY_VERSION > '1.8'
-    # requires ruby 1.9+, on 1.8 we'll fall back to the old regex parsing
-    gem 'rspec-json_expectations', '~> 1.4'
```

Comment:
Because @binford2k was nice enough to politely skip that when the module doesn't load with a Rescue statement.